x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/feature_table.tsx

```diff
@@ -267,9 +275,14 @@ export class FeatureTable extends Component<Props, State> {
 
     if (feature.reserved && primaryFeaturePrivileges.length === 0) {
       const buttonContent = (
-        <>
-          {<EuiIcon type="empty" size="l" />} <FeatureTableCell feature={feature} />
-        </>
+        <EuiFlexGroup gutterSize="xs" direction="row">
+          <EuiFlexItem grow={false}>
+            <EuiIcon type="empty" size="l" />
```

Comment:
Is this empty icon being used to align the text for features without an accordion to feature text that does have an accordion? If so, would it be better to use padding or margins to achieve the desired indentation, rather than adding additional elements to the page (`EuiFlexItem`, `EuiIcon`, etc.)?

Reply:
I'm not too sure what the original intent was either, since I just moved this piece of existing code around. I suspect it is doing what you think too.